documentation/modules/ROOT/partials/modules/all-connectors/con-connector-ad-hoc-snapshots.adoc

```diff
@@ -88,3 +91,11 @@ For example:
 INSERT INTO myschema.debezium_signal VALUES('ad-hoc-1', 'execute-snapshot', '{"data-collections": ["schema1.table1", "schema2.table2"]}')
 ----
 ////
+
+.Triggering an ad hoc blocking snapshot
```

Comment:
I agree that `on-demand` is more appealing but we need to consider that we always used `ad-hoc` also for the incremental snapshot. In case we want rename we need to consider it.

Reply:
https://knowledgeincreasesbysharing.quora.com/Is-adhoc-and-on-demand-are-the-same-word

Reply:
At least in English, these two terms have different meanings; although it's quite common for people to use one or the other incorrectly because the differences between them are subtle.

I think in the context of Debezium, users rely on incremental snapshots for both ad-hoc and on-demand reasons, and the subtle difference depends entirely on why a user is performing the incremental snapshot.

1. "Oh a row was missed by Debezium due a crash, let me use incremental snapshots to fix the problem".   
In this context, it's valid to use the term `ad-hoc` because its one-off.
2. "Oh I don't want to wait for initial snapshots, let me do it concurrently while streaming". 
In this context, it's valid to use the term `on-demand`.

That said, I'm fine with the general consensus because I understand that these subtle differences don't always translate to non-native speakers so whatever the more widely accepted terminology is, would likely be the best choice.

Reply:
I'd keep the `ad-hoc` name for time beiing. it is in place for quite some time and I don't see added value.
I'd defer to @roldanbob as the final arbiter as something tells me that he his the one who wields the language best :-)
But if the rename is going to be done then it should be a separate issue and all done as a single change.

Reply:
@roldanbob If you agree we can merge this. About the renaming, you got the ball now :-)

Reply:
@mfvitale @jpechane Although I said that I'd support a change in terminology, I don't see an urgent need for us to make that change. As @jpechane states, the current usage has been in place for some time now, and it hasn't broken anything yet, so I'm fine with leaving it as-is.